appveyor.yml

```diff
@@ -43,6 +43,8 @@ build_script:
   - travis-tool.sh r_binary_install curl
   - Rscript -e "install.packages('covr', repos = 'http://cran.us.r-project.org')"
   - Rscript -e "install.packages('https://ci.appveyor.com/api/projects/open-systems-pharmacology-ci/ospsuite-rutils/artifacts/ospsuite.utils.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "install.packages(c('ggplot2', 'patchwork', 'vdiffr'), repos = 'http://cran.us.r-project.org')"

```

Comment:
`covr `could be included here in the list of installed packages - then line 44 could be removed

Reply:
I am _deliberately_ making a distinction between package dependencies (hard or soft) and packages (like _covr_ or _lintr_) needed only for the CI/CD.